tests/UI/data/faker/message.ts

```diff
@@ -37,6 +39,9 @@ export default class MessageData {
     /** @type {string} Firstname of the customer */
     this.lastName = messageToCreate.lastName || faker.name.lastName();
 
+    /** @type {string} employee to forward the message */
+    this.employee = messageToCreate.employee || `${this.firstName.slice(0.1)}. ${this.lastName}`;
```

Comment:
```suggestion
    /** @type {string} employee to forward the message */
    this.employeeName = messageToCreate.employee || `${this.firstName.slice(0.1)}. ${this.lastName}`;
```

I think you can rename the property or change the property with a EmployeeData type.

Reply:
Done :+1: